sdk/servicebus/service-bus/test/utils/managementUtils.ts

```diff
@@ -9,6 +9,9 @@ import {
   delay
 } from "../../src";
 
+import * as dotenv from "dotenv";
+dotenv.config();
```

Comment:
The functions exported from this file are used either in `atomManagement.spec.ts` file or in `testUtils.ts` file, both of which already make a call to `dotenv.config()`. What is the reason for us to do the same again here?

Reply:
I thought the same too, but this change was needed as the environment variables were not being loaded otherwise.

Reply:
That could be because of the `const env = getEnvVars()` line that is outside of the functions.
If we move it inside `getManagementClient()` which is the only place where it is used, then we shouldnt need the above

Reply:
Agree, updated